packages/xml-views-completion/src/providers/attributeValue/namespace.ts

```diff
@@ -29,7 +29,7 @@ export function namespaceValueSuggestions(
 
   const ui5Model = opts.context;
 
-  const attributeValue = xmlAttribute.value ?? "";
+  const attributeValue = opts.prefix ?? "";
```

Comment:
can we have this in a different PR? 
This is a separated fix right?

I think we will lose CHANGELOG info
once this is squashed

Reply:
Ok... I did it in the same PR because I found the bug while I was writing the tests. I can put the tests in skip and re-enable them in PR with this fix.

Reply:
Done